app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value():
-    pass
+    assert sum(split_integer(33, 5)) == 33
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    assert split_integer(10, 2) == [5, 5]
 
 
 def test_should_return_part_equals_to_a_value_when_slitting_into_one_part():
-    pass
+    assert split_integer(17, 4) == [4, 4, 4, 5]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal():
-    pass
+    assert sorted(split_integer(17, 4)) == [4, 4, 4, 5]
```

Comment:
you don't need `sorted` here because `split_integer` returns already sorted list